tests/feature_point_id/test_feature_point_id.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import cv2 as cv
+from unittest.mock import patch
+from modules.feature_point_id.feature_point_id import FeaturePointId
+from database.descriptors_db import DescriptorsDB
+from database.db_protocol import DBProtocol
+
+
+class FeaturePointId_get_with_pixel(unittest.TestCase):
+    def setUp(self):
+        DBProtocol("test_descriptors", "(id INT AUTO_INCREMENT, descriptor JSON, PRIMARY KEY (id))").delete_all()
+        
+    def tearDown(self):
+        DBProtocol("test_descriptors", "(id INT AUTO_INCREMENT, descriptor JSON, PRIMARY KEY (id))").delete_all()
+
+    def test_new_feature_point_id(self):
+        with patch.dict("os.environ", {"DESCRIPTORS_DB_TABLE": "test_descriptors"}):
+            frame = cv.imread("tests/feature_point_id/sample.jpg")
+            featurePointIdclass = FeaturePointId()
+            db = DescriptorsDB()
+            response = featurePointIdclass.get_with_pixel(frame)
+            expected = len(db.get_all())
+            self.assertEqual(expected, len(response))
+    
+    def test_known_feature_point_id(self):
+        with patch.dict("os.environ", {"DESCRIPTORS_DB_TABLE": "test_descriptors"}):
+            frame = cv.imread("tests/feature_point_id/sample.jpg")
+            featurePointIdclass = FeaturePointId()
+            db = DescriptorsDB()
+            response = featurePointIdclass.get_with_pixel(frame)
+            expected = len(db.get_all())
+            self.assertEqual(expected, len(response))
+            response = featurePointIdclass.get_with_pixel(frame)
+            expected = len(db.get_all())
+            self.assertEqual(expected, len(response))

```

Comment:
> もう一度改めてDBにget_allしてDBが変わらないことを見ようとしてる

あ、確かに！
細かい心配だけど、今の実装だと万が一、dbが更新されてresponceにもdbにあるすべてのidが紐付けされたときに気がつけない気がする。
この書き方どう？
```suggestion
            response_first = featurePointIdclass.get_with_pixel(frame)
            expected_first = len(db.get_all())
            self.assertEqual(expected, len(response))
            response_second = featurePointIdclass.get_with_pixel(frame)
            expected_second = len(db.get_all())
            self.assertEqual(expected_first, expexted_second) # もしget_with_pixelにバグがあった場合のために
            self.assertEqual(expected, len(response))
```

Reply:
いったんこのMRはこれでmergeして、上の議論ふまえてもう一度test書く

Reply:
そうしよう！